docs/content/en/docs/install/install.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Install KLT
+description: Install the Keptn Lifecycle Toolkit
+icon: concepts
+layout: quickstart
+weight: 35
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+Two methods are supported for installing the Keptn Lifecycle Toolkit:
+
+* Releases 0.7.0 and later can be installed using
+  the [Helm Chart](#use-helm-chart).
+  This is the preferred strategy because it allows you to customize your cluster..
+
+* All releases can be installed using
+  the [manifests](#use-manifests).
+  This is the less-preferred way because it does not support customization.
+
+## Use Helm Chart
+
+Version 0.7.0 and later of the Lifecycle Toolkit
+should be installed using Helm Charts.
+The command sequence to fetch and install the latest release is:
+
+```shell
+helm repo add klt https://charts.lifecycle.keptn.sh
+helm repo update
+helm upgrade --install keptn klt/klt \
+   -n keptn-lifecycle-toolkit-system --create-namespace --wait
+```
+
+Note that the `helm update` command is used for fresh installs
+as well as for upgrades.
+
+Use the `--version <version>` flag on the
+`helm upgrade --install` command line to specify a different KLT version..
+
+Use the following command sequence to see a list of available versions:
+
+```shell
+helm repo update
+helm search repo keptn-lifecycle-toolkit
+```
+
+To modify configuration options, use the `--set` flag
+to the `helm upgrade --install` command.
+Configuration options are specified using the format:
+
+```shell
+key1=value1,key2=value2,....
+```
```

Comment:
I would put this section below the values section. It should be less prominent since it's not a best practise way of customizing helm charts.
Also:
```suggestion
To modify configuration options, use the `--set` flag
to the `helm upgrade --install` command.
Configuration options are specified using the format:

```shell
--set key1=value1,key2=value2,....
```
```